src/typeahead/typeahead.ts

```diff
@@ -72,7 +72,8 @@ let nextWindowId = 0;
     'autocapitalize': 'off',
     'autocorrect': 'off',
     'role': 'combobox',
-    'aria-autocomplete': 'list',
+    'aria-multiline': 'false',
```

Comment:
We don't seem to have a test for the `aria-multiline`. Could you please add one?